v2/guides/guides.mdx

```diff
@@ -1,20 +1,24 @@
 ---
 id: guides
-title: Recipe guides
+title: User Recipes
```

Comment:
When you toggle the select it defaults to "Select Recipe" instead of keeping its value